doc/source/examples/multiphysics/melting-cavity/melting-cavity.rst

```diff
@@ -0,0 +1,235 @@
+==========================
+Melting Cavity
+==========================
+
+This example simulates a `two-dimensional gallium melting cavity`_. 
```

Comment:
Maybe add the reference to the Blais & Al paper here to say where you have taken this benchmark :).